src/github.com/getlantern/autoupdate-server/server/github.go

```diff
@@ -272,12 +297,16 @@ func (g *ReleaseManager) pushAsset(os string, arch string, asset *Asset) (err er
 	if g.latestAssetsMap[os] == nil {
 		g.latestAssetsMap[os] = make(map[string]*Asset)
 	}
-	if g.latestAssetsMap[os][arch] == nil {
-		g.latestAssetsMap[os][arch] = asset
-	} else {
-		// Compare against already set version
-		if asset.v.GT(g.latestAssetsMap[os][arch].v) {
+
+	// Only considering non-manoto versions for the latestAssetsMap
+	if !buildStringContainsManoto(asset.v) {
```

Comment:
Yeah, we'll have to make it 2.0.0+stable. We won't have any problem as long as we only keep +stable and +manoto, +manoto is ignored by us and +tags are ignored by semver.